notebooks/Readme.md

```diff
@@ -0,0 +1,27 @@
+## Notebook Examples
+
+In this folder you can find [jupyter notebook](https://jupyter.org/) examples to illustrate how to automate certain aspects of, and around ilastik from Python.
+
+The structure in each folder will follow the same scheme:
+
+* notebooks (here)
```

Comment:
Maybe show the directory structure with the output of the `tree` utility?

Reply:
Yeah, okay I will update that - not too happy with the list either, cheers.